plugins/serializers/registry.go

```diff
@@ -103,6 +103,9 @@ type Config struct {
 	// Output string fields as metric labels; when false string fields are
 	// discarded.
 	PrometheusStringAsLabel bool `toml:"prometheus_string_as_label"`
+
+	// Output as text or protobuf
+	PrometheusRemoteWriteFormat bool `toml:"prometheus_remote_write_format"`
```

Comment:
Seems like this would be better off as a string of either "text" or "remote_write", and maybe a generic name like WriteFormat

Reply:
No, I forgot to remove it, because before use boolean, I wrote code with string config.

Reply:
PR updated with a string configuration for prometheus write format.